handbook/ce/index.md

```diff
@@ -60,9 +60,13 @@ CSEs are the go-to technical team for our CEs, helping customers both pre- and p
   - [Mike McLaughlin](../../company/team/index.md#mike-mclaughlin-he-him)
   - [Alex Fogg](../../company/team/index.md#alex-fogg-he-him)
   - T. (Director of Customer Engineering, US East; starting 2021-03-01)
-- Customer Support
+- Customer Support Engineers
+  - Adeola Akinsiku
+  - [Warren Gifford](../../company/team/index.md#giselle-northy-she-her)
```

Comment:
Looks like the markdown links for Warren and Giselle are not pointing to the correct profiles.

Reply:
Oh shoot good catch!

Reply:
Thanks for catching that, Giselle! It's like I was trying to make a point about why PR and I wasn't 🤣 .... but also totally why a second set of 👀 is gold ✨